extensions/amp-list/0.1/amp-list.js

```diff
@@ -1003,6 +1022,9 @@ export class AmpList extends AMP.BaseElement {
         return this.mutateElement(() => {
           if (this.loadMoreSrc_) {
             this.getLoadMoreService_().toggleLoadMoreLoading(false);
+            if (lastTabbableChild && opt_fromClick) {
+              lastTabbableChild./*REVIEW*/ focus();
```

Comment:
Import `tryFocus` from `dom.js` and use that instead. The REVIEW here is just because we want to handle API exceptions at the runtime level, instead of in every extension.